Resources/Prototypes/Entities/Objects/Devices/pda.yml

```diff
@@ -633,7 +633,7 @@
   parent: BasePDA
   id: HoSPDA
   name: head of security PDA
-  description: Whosoever bears this PDA is the law.
+  description: Smells like donuts and gunpowder residue.
```

Comment:
Why?

Reply:
Not my idea. This is because the description "Whosoever bears this PDA is the law" is given to the Chief Justice PDA instead of HoS, so this is its replacement. Personally, I think it fits Chief Justice more

Reply:
If you want to rename files, it has to be done via locale in a delta folder.
i.e
![image](https://github.com/DeltaV-Station/Delta-v/assets/142105406/757df7e4-0461-41b9-ba4b-53d9b34bb1cd)
![image](https://github.com/DeltaV-Station/Delta-v/assets/142105406/863fc98d-00e7-4d7f-b254-ef5a08076877)